sale_order_price_recalculation/tests/test_sale_order_price_recalculation.py

```diff
@@ -78,7 +78,8 @@ def test_price_recalculation(self):
         self.sale_order_line.name = "My product description"
         self.assertEqual(self.sale_order_line.price_unit, self.product.lst_price)
         # Change price
-        self.product.lst_price = 500
+        with Form(self.product) as product:
+            product.lst_price = 500
```

Comment:
naive question. Why this change ?

Reply:
Direct update was not calling price_compute of product, hence price not set to 500 in sale order line and test failing

Reply:
oki, thanks